beacon-chain/cache/depositsnapshot/spec_test.go

```diff
@@ -222,9 +224,9 @@ func merkleRootFromBranch(leaf [32]byte, branch [][32]byte, index uint64) [32]by
 	for i, l := range branch {
 		ithBit := (index >> i) & 0x1
 		if ithBit == 1 {
-			root = sha256.Sum256(append(l[:], root[:]...))
+			root = hash.Hash(append(l[:], root[:]...))
 		} else {
-			root = sha256.Sum256(append(root[:], l[:]...))
+			root = hash.Hash(append(root[:], l[:]...))
```

Comment:
I wasn't sure but is this possibly a loop reference issue? instead of doing branch[i][:] 
maybe not if it's using [:] but double-checking

Reply:
Since we are appending , we will be copying it here. So it will not be an issue